Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/App.xaml.cs

```diff
@@ -74,17 +74,6 @@ protected override void OnStart()
                 {
                     AppCenterLog.Info(LogTag, "Crashes.LastSessionCrashReport.Exception=" + task.Result?.Exception);
                 });
-                Auth.SignInAsync().ContinueWith(task =>
```

Comment:
I suppose this was intended to be removed pending the official UI work?

Reply:
It was moved to a button in others page.